### ISSUE TITLE
 ktlint 0.47.1 displays plenty of warnings about the --disabled_rules argument usage. `ktlint_disabled_rules` does not get picked up. 

### ISSUE THREAD

**Issue by amit-nayar** [pinterest/ktlint#1709]
Related to  #1599 

## Expected Behavior
When I pass the rules I want to disable in my spotless-ktlint gradle setup using `editorConfigOverride`, `disabled_rules` does not produce warnings, or using `ktlint_disabled_rules` works:

### Using `disabled_rules`

```
    spotless {
        kotlin {
            target '**/*.kt'
            ktlint(Versions.BuildSystem.KtLint).editorConfigOverride([
                    'disabled_rules' : '<rule_name>',
            ])
        }
    }
```

Produces warnings.

### Using `ktlint_disabled_rules`

```
    spotless {
        kotlin {
            target '**/*.kt'
            ktlint(Versions.BuildSystem.KtLint).editorConfigOverride([
                    'ktlint_disabled_rules' : '<rule_name>',
            ])
        }
    }
```

The map entry is not picked up and the rules are not disabled.

## Observed Behavior
When I try to apply `disabled_rules` to `ktlint` using `spotless`, I get many warnings:

```
Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
```

If I replace with the recommended `ktlint_disabled_rules`, it does not get picked up and the rules are not disabled.

## Steps to Reproduce
Using the `spotless` plugin in gradle (I am using current latest `6.11.0`) the relevant code is under "Expected Behavior" heading.

## Your Environment
- ktlint: 0.47.1
- spotless: 6.11.0
- android gradle plugin: 7.1.2

**Comment by jjbr** [comment]
For reference, this seems to be fixed in #1671, so should be available as of the next release.

**Comment by amit-nayar** [comment]
Excellent!
